BedrockServer.cpp

```diff
@@ -1676,6 +1676,7 @@ void BedrockServer::_status(unique_ptr<BedrockCommand>& command) {
         content["version"] = _version;
         content["host"] = args["-nodeHost"];
         content["commandCount"] = BedrockCommand::getCommandCount();
+        content["detached"] = isDetached() ? "true" : "false";
```

Comment:
```suggestion
        content["isDetached"] = isDetached() ? "true" : "false";
```

This will follow our naming conventions